.gitmodules

```diff
@@ -22,3 +22,18 @@
 [submodule "vendor/avb"]
 	path = vendor/avb
 	url = https://android.googlesource.com/platform/external/avb
+[submodule "vendor/libbase"]
+	path = vendor/libbase
+	url = https://android.googlesource.com/platform/system/libbase
+[submodule "vendor/libziparchive"]
+	path = vendor/libziparchive
+	url = https://android.googlesource.com/platform/system/libziparchive
+[submodule "vendor/base"]
+	path = vendor/base
+	url = https://android.googlesource.com/platform/frameworks/base
+[submodule "vendor/native"]
+	path = vendor/native
+	url = https://android.googlesource.com/platform/frameworks/native
+[submodule "compat/gsl"]
+	path = compat/gsl
+	url = https://github.com/microsoft/GSL.git
```

Comment:
Could you briefly explain why this is needed?

Reply:
This library was added to provide fallback implementation for `std::span` in gcc 9 and prior.
Historically, `span` was implemented on Microsoft GSL first, then propose to C++ LWG later. (And it's removed from GSL after it's accepted to C++20)

Reply:
Since gsl::span was removed from current GSL, we can just vendor `gsl::span` in.

Reply:
So with `gcc >= 10` this is not required?

Reply:
> So with `gcc >= 10` this is not required?

Yes, that's correct. `std::span` is avaiable with gcc 10 and clang 7+